draft-spaghetti-grow-nrtm-v4.xml

```diff
@@ -717,11 +717,15 @@
   "changes": [
     {
       "action": "delete",
-      "existing_object": "route: 192.0.2.0/24\norigin: AS65530\nsource: EXAMPLE"
```

Comment:
Action should be enough to tell us if it is a new object or existing object.

Reply:
This has been resolved, the element name is now "object"